packages/cozy-konnector-libs/src/libs/solveCaptchaWithFullResponse.js

```diff
@@ -0,0 +1,179 @@
+/**
+ * Use every possible means to solve a captcha. At the moment, Anticaptcha web service is used if
+ * any related secret key is found in COZY_PARAMETERS environment variable.
+ *
+ * @module solveCaptchaWithFullResponse
+ */
+const log = require('cozy-logger').namespace('solveCaptchaWithFullResponse')
+const errors = require('../helpers/errors')
+const request = require('request-promise')
+const sleep = require('util').promisify(global.setTimeout)
+
+const connectorStartTime = Date.now()
+const ms = 1
+const s = 1000 * ms
+const m = 60 * s
+
+const DEFAULT_TIMEOUT = connectorStartTime + 3 * m // 3 minutes by default to let 1 min to the connector to fetch files
+
+/**
+ * Use every possible means to solve a captcha. At the moment, Anticaptcha web service is used if
+ * any related secret key is found in COZY_PARAMETERS environment variable.
+ * If you do not want to solve the captcha each time the connector is run, please also use
+ * CookieKonnector which will help you save the session.
+ *
+ * Parameters:
+ *
+ * - `params` is an array of objects with any attributes with some mandatory attributes :
+ *   + `type` (String): (default recaptcha) type of captcha to solve. can be "recaptcha" or "image" at the moment
+ *   + `timeout` (Number): (default 3 minutes after now) time when the solver should stop trying to
+ *   solve the captcha
+ *   + `websiteKey` (String): the key you can find on the targeted website (for recaptcha)
+ *   + `websiteURL` (String): The URL of the page showing the captcha (for recaptcha)
+ *   + `body` (String): The base64 encoded image (for image captcha)
+ * Returns: Promise with the solved captcha response as a string
```

Comment:
nice doc, you should definitly write that within the JSDoc format! It'll be useful for our IDE.